src/BeastConfig.h

```diff
@@ -0,0 +1,165 @@
+//------------------------------------------------------------------------------
```

Comment:
Agreed, it would be good to remove.  I'm unsure who you're getting your marching orders from, or how much longer your internship lasts.  If you have time and permission, then I think either removing or reducing and renaming BeastConfig.h would be a nice step forward.  But it would want to be in a separate pull request.